### PR TITLE
RRK data layer picker (5 pillars)

### LINKED ISSUE #68: Map control panel - RRK data layer picker

### PR DESCRIPTION
**Pull request by thequeenofspades**
## Changes
- Adds controls to the frontend UI for toggling on/off metrics for each of the 5 pillars with available raster data ("biodiversity", "carbon_sequestration","forest_resilience", "fire_dynamics", "fire_adapted_communities")
- Includes a checkbox to toggle on data normalization
- Updates unit tests (fixes test failure from previous change to map count select)
- Remove previous data layers (no longer needed)
- Ran file formatter
- Closes #68 (for initial 5 pillars)

image

## Todo
- Improve UX for viewing/selecting condition metrics (pillars should be collapsible)
- Grey out / visually indicate unavailable metrics (blocked on #158 )

### CODE REVIEW

File: src/interface/src/app/map/map.component.ts [hunk 1]
```diff
@@ -708,6 +634,33 @@ export class MapComponent implements AfterViewInit, OnDestroy {
     }
   }
 
+  changeConditionsLayer(map: Map) {
```
Comment (lattemilk) [inline src/interface/src/app/map/map.component.ts:643]: Add docstring to public methods? :D

**Review by lattemilk — APPROVED** [review]

File: src/interface/src/app/app.component.scss [hunk 2]
```diff
@@ -7,5 +7,5 @@
 }
 
 app-navigation {
-  height: 100%;
+  height: calc(100% - 48px);
```
[the remark this answers is not in the document]
Comment (thequeenofspades) [reply src/interface/src/app/app.component.scss:10]: Good idea! Created a shared variables scss file.

File: src/interface/src/app/map/map.component.ts [hunk 3]
```diff
@@ -1,32 +1,19 @@
...
 import { MapService } from '../map.service';
-import { PlanCreateDialogComponent } from './plan-create-dialog/plan-create-dialog.component';
 import { PlanService, PlanState } from '../plan.service';
 import { PopupService } from '../popup.service';
 import { SessionService } from '../session.service';
-import { BaseLayerType, DataLayerType, Map, MapConfig, Region } from '../types';
+import { BaseLayerType, ConditionsConfig, defaultMapConfig, Map, Region } from '../types';
 import { Legend } from './../shared/legend/legend.component';
+import { PlanCreateDialogComponent } from './plan-create-dialog/plan-create-dialog.component';
 import { ProjectCardComponent } from './project-card/project-card.component';
```
[the remark this answers is not in the document]
Comment (thequeenofspades) [reply src/interface/src/app/map/map.component.ts:24]: I was thinking the same. map.component.ts is getting messy. I can work on refactoring it.

File: src/interface/src/app/map/map.component.ts (hunk 1 above)
Comment (thequeenofspades) [reply src/interface/src/app/map/map.component.ts:644]: Done!

**Pull request merged**